haystack/components/readers/extractive.py

```diff
@@ -5,7 +5,7 @@
 
 from haystack import ComponentError, Document, ExtractedAnswer, component, default_from_dict, default_to_dict, logging
 from haystack.lazy_imports import LazyImport
-from haystack.utils import ComponentDevice, DeviceMap
+from haystack.utils import ComponentDevice, DeviceMap, Secret, deserialize_secrets_inplace
```

Comment:
do we need release notes for this change?

Reply:
It would be better.

[This](https://github.com/deepset-ai/haystack/blob/c1c0cbfde462ff5490731b4425b3c8e51b45218e/releasenotes/notes/update-secret-handling-in-components-925d4f3c3c9530db.yaml) can serve as inspiration.

Reply:
It's a breaking change so I'd say yes. We also need to check tutorials/cookbooks using the ExtractiveReader.

Reply:
I just checked. We're not using private models in those so no updates needed: https://haystack.deepset.ai/tutorials/34_extractive_qa_pipeline

Reply:
thanks!

Reply:
added the release note